tests/testthat/_snaps/arg.md

```diff
@@ -5,25 +5,29 @@
     Output
       <error/rlang_error>
       `"continuuos"` must be one of "discrete" or "continuous".
-      Did you mean "continuous"?
+      x Input is "continuuos".
```

Comment:
In this case, I think we could add it to the end of the first sentence:

`"continuuos"` must be one of "discrete" or "continuous"; not "continuuos"

And maybe rework the tests so they always use a variable? (Since that will be the case you actually see in real life):

`x` must be one of "discrete" or "continuous"; not "continuuos"

Reply:
@batpigandme might have thoughts on the punctuation here.

Reply:
Both done. A google search about suggests using a comma rather than a semi-colon is the correct punctuation.